### PR TITLE
Update arrow, netty and other dependencies 

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Comment by wendigo** [comment]
PTAL @hashhar @ebyhr

**Comment by wendigo** [comment]
ping @hashhar @ebyhr

File: plugin/trino-bigquery/pom.xml [hunk 1]
```diff
@@ -279,6 +279,12 @@
         <dependency>
             <groupId>org.apache.arrow</groupId>
             <artifactId>arrow-vector</artifactId>
+            <exclusions>
+                <exclusion>
```
Comment (hashhar) [inline plugin/trino-bigquery/pom.xml:283]: a comment about what it conflicts with helps when updating versions in future or removing exclusions in future

File: pom.xml [hunk 2]
```diff
@@ -263,6 +262,14 @@
                 <scope>import</scope>
             </dependency>
 
+            <dependency>
```
Comment (hashhar) [inline pom.xml:265]: nice

**Review by hashhar — APPROVED** [review]

**Pull request merged**